inetum-realjobs-frontend/src/app/components/resume/details/language-list/language-list.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {Language} from "../../../../models/language";
+import {SkillLevel} from "../../../../models/skillLevel.enum";
+
+@Component({
+  selector: 'app-language-list',
+  templateUrl: './language-list.component.html',
+  styleUrls: ['./language-list.component.scss']
+})
+export class LanguageListComponent implements OnInit {
+
+  @Input() languages: Language[];
+  @Input() deleteButton: boolean;
+  @Output() deleteEvent = new EventEmitter<number>();
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
+  }
```

Comment:
Empty `ngOnInit` should be removed. Don't forget to remove `implements OnInit` for the class.